app/services/handlers/user/get-user-handler.js

```diff
@@ -0,0 +1,36 @@
+// app/services/handlers/user/get-user-handler.js
+import Service, { inject as service } from '@ember/service';
+
+export default class HandlersUserGetUserHandler extends Service {
```

Comment:
Should not be called `.../getUsersHandler` ? As we normalize array of users with this handler

Reply:
Yep makes sense - Done 👍🏼

Reply:
Changed here: https://github.com/Mr-Smyth/task-manager/pull/17/commits/f2fd2326a3a1356127074befebe6c482ea0c274d